packages/react-form-state/docs/validators.md

```diff
@@ -23,6 +23,8 @@ import {validators} from '@shopify/react-form-state';
 
 We can pass these directly into the `validators` prop of `<FormState />`.
 
+To validate the form automatically on form submit set the `validateOnSubmit` prop of `FormState` to be `true`.
```

Comment:
```suggestion
To use validators when submitting your form, set the `validateOnSubmit` prop of `<FormState />` to `true`. Here's an example:
```

I would also remove the line break to make a paragraph out of the sentence above, and then edit the example below to use this property.